pkg/validation-webhook/snapshot.go

```diff
@@ -30,8 +31,12 @@ import (
 var (
 	// SnapshotV1Beta1GVR is GroupVersionResource for volumesnapshots
 	SnapshotV1Beta1GVR = metav1.GroupVersionResource{Group: volumesnapshotv1beta1.GroupName, Version: "v1beta1", Resource: "volumesnapshots"}
+	// SnapshotV1GVR is GroupVersionResource for volumesnapshots
```

Comment:
how about make the comments more specific, or remove them as the names already suggested?
// SnapshotV1GVR is GroupVersionResource for v1 VolumeSnapshots
// SnapshotV1GVR is GroupVersionResource for v1beta1 VolumeSnapshots
// // SnapshotContentV1Beta1GVR is GroupVersionResource for v1beta1 VolumeSnapshotContents
etc.